src/Components/Modal/EditMovie.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Button, Modal,Form } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import './style.scss';
+
+export default function EditMovie(props) {
```

Comment:
there are a lot of similarities in add and edit movie modals. Maybe you can make it one modal? or create separate component that'll aggregate all of this similarities and them reuse it in this both modals (probably'll be the better solution)